### PR TITLE
Update dependencies. Vega 4.1

### PR DESCRIPTION
**Pull request by domoritz**

### CODE REVIEW

**Comment by domoritz** [comment]
@kanitw Why does this not rebuild the SVGs?

**Comment by kanitw** [comment]
Well the push Travis isn't run -- probably due to GitHub failure.

**Comment by domoritz** [comment]
@kanitw and now?

**Comment by domoritz** [comment]
I already rebased.

**Comment by domoritz** [comment]
@kanitw infinite loop?

File: .travis.yml [hunk 1]
```diff
@@ -4,7 +4,6 @@ addons:
 
 language: node_js
 node_js:
-  - "6"
```
Comment (domoritz) [inline .travis.yml]: @kanitw can we revert this?

**Pull request merged**